mlflow/server/handlers.py

```diff
@@ -1776,6 +1785,22 @@ def _create_registered_model():
         description=request_message.description,
     )
     response_message = CreateRegisteredModel.Response(registered_model=registered_model.to_proto())
+
+    try:
+        dispatch_webhook(
+            event=WebhookEvent.REGISTERED_MODEL_CREATED,
+            payload=RegisteredModelCreatedPayload(
+                name=request_message.name,
+                tags={t.key: t.value for t in request_message.tags},
+                description=request_message.description,
+            ),
+            store=_get_model_registry_store(),
+        )
+    except Exception as e:
```

Comment:
Let's move the try catch inside dispatch_webhook?

Reply:
That would make `dispatch_webhook` fail silently.

Reply:
We can still warn inside dispatch_webhook

Reply:
I think the current version of the code makes the intent clearer:


```python
dispatch_webhook(...)
```

`dispatch_webhook` might fail. We need to see what `dispatch_webhook` does.

vs.

```python
try:
    dispatch_webhook(...)
except Exception:
    ...
```

Cool, we ignore the error `dispatch_webhook` throws

Reply:
I'm fine with either, I would put it inside purely because I don't want to try catch everywhere :)